pandas/tests/extension/test_external_block.py

```diff
@@ -5,7 +5,8 @@
 
 import pandas as pd
 from pandas.core.internals import (
-    BlockManager, SingleBlockManager, NonConsolidatableMixIn, Block)
+    BlockManager, SingleBlockManager)
+from pandas.core.internals.blocks import Block, NonConsolidatableMixIn
```

Comment:
For next PRs doing further splitting, can we please keep the existing imports? 

What I mean is: import all the things that are used in pandas outside of internals itself in `internals/__init_.py`, then no changes to imports are needed, and it is also clear which functions/classes are used in the rest of pandas.

Reply:
> For next PRs doing further splitting, can we please keep the existing imports?

Yes.  I'm also marking which non-internals modules are importing what, in the hopes we can trim down the exposed surface.